RadarSDK/RadarUser.h

```diff
@@ -141,10 +142,20 @@ typedef NS_ENUM(NSInteger, RadarLocationSource);
 @property (assign, nonatomic, readonly) RadarLocationSource source;
 
 /**
- A boolean indicating whether the user's IP address is a known proxy. May be `false` if Fraud is not enabled.
+ `true` if the user's IP address is a known proxy. May be `false` if Fraud is not enabled.
  */
 @property (assign, nonatomic, readonly) BOOL proxy;
 
+/**
+ `true` if` the user's IP address is being mocked, such as in a simulation. May be `false` if Fraud is not enabled.
+ */
+@property (assign, nonatomic, readonly) BOOL mocked;
+
+/**
+ If Fraud is enabled, this will be non-`nil`, and will indicate whether the user's IP address or location is being mocked or proxied.
```

Comment:
Let's follow what we have for the `insights` object here: "Learned fraud state for the user. May be `nil` if Fraud is not enabled" - we'll eventually want to represent things beyond just mocked / proxied

Reply:
Done.